### PR TITLE
add node types: input and output

### PR DESCRIPTION
**Pull request by dstang2000**
I have add input / output node which is trapezoid in shape.
image

### CODE REVIEW

**Comment by adrai** [comment]
What does tds mean?

**Comment by adrai** [comment]
can you also extend the readme?

File: src/flowchart.symbol.input.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    x: this.textMargin * 3
+  });
+
+  var width = this.text.getBBox().width + 4 * this.textMargin;
+  var height = this.text.getBBox().height + 2 * this.textMargin;
+  var startX = this.textMargin;
+  var startY = height/2;
+
+  var start = {x: startX, y: startY};
+  var points = [
```
Comment (adrai) [inline src/flowchart.symbol.input.js:21]: The points seems to be the same as the output.js...
I think that should be changed to:

```js
var points = [
    {x: startX - this.textMargin + 2 * this.textMargin, y: height},
    {x: startX - this.textMargin + width, y: height},
    {x: startX - this.textMargin + width + 2 * this.textMargin, y: 0},
    {x: startX - this.textMargin, y: 0},
    {x: startX, y: startY}
  ];
```

**Comment by dstang2000** [comment]
Ok, I have updated the readme.md  (and update example.svg, add input.png, output.png)

**Comment by adrai** [comment]
thank you... have you seen my comment here: https://github.com/adrai/flowchart.js/pull/251#discussion_r1420775118 ?

File: src/flowchart.symbol.input.js (hunk 1 above)
Comment (dstang2000) [reply src/flowchart.symbol.input.js:21]: my local file like this:
`var points = [
    {x: startX - this.textMargin + 2 * this.textMargin, y: height},
    {x: startX - this.textMargin + width, y: height},
    {x: startX - this.textMargin + width + 2 * this.textMargin, y: 0},
    {x: startX - this.textMargin, y: 0},
    {x: startX, y: startY}
  ];
`

Comment (dstang2000) [reply src/flowchart.symbol.input.js:21]: now input.js is pushed. please check. Thank you!

**Comment by adrai** [comment]
just release with v1.18.0

**Comment by dstang2000** [comment]
> just release with v1.18.0

adrai, Thank you very much!
I have update my project [dstang2000/py2flowchart](https://github.com/dstang2000/py2flowchart) and  vscode extension [dstang2000/vscode-py2flowchart](https://github.com/dstang2000/vscode-py2flowchart) which use the flowchart.js to convert python code to flowchart.

PS: maybe a section can be added into README.md to describe what projects are using the flowchart.js @adrai

**Pull request merged**